src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int Number_OfBalls = 3;
+        Ball[] balls = new Ball[Number_OfBalls];
+        for (Ball ball : balls) {
+            Lottery lottery = new Lottery();
+            ball = lottery.getRandomBall();
```

Comment:
does this operation (`ball = lottery.getRandomBall();`) set new ball to array of `balls `?